tests/test_management_commands.py

```diff
@@ -59,6 +162,75 @@ def test_basic_invocation(self):
         first_result = results[0]
         self.assertEqual(str(movie.id), first_result.meta.id)
 
+    def test_newer_flag(self):
+        """
+        Test that when two versions of the "movies" index are available,
+        and the older one is activated, `./manage.py es_update movies --newer`
+        indexes into the newer, unactivated version
+        """
+        index_model, version_model, dem_index = self.check_basic_setup_and_get_models()
+
+        # create two newer versions of the movies index
+        call_command(*"es_create movies --force".split())
```

Comment:
I like this method of passing in commands.  I'm going to start doing this now as it makes it easier to copy and paste the command for later debuging/testing.